applications/jupyter/workloads.tfvars

```diff
@@ -13,14 +13,15 @@
 # limitations under the License.
 
 ##common variables
-## Need to pull this variables from tf output from previous platform stage
+## Need to pull this variables from tf output from previous infrastructure stage
 project_id = "<your project ID>"
 
 ## this is required for terraform to connect to GKE master and deploy workloads
-create_cluster        = false # this flag will create a new standard public gke cluster in default network
+create_cluster        = false # this flag will create a new autopilot public gke cluster in the default network
```

Comment:
It doesn't depend on that variable. If this is false then it doesn't matter what the below variables are because no cluster will be created. I've updated the comment though to make it clearer. PTAL.